core/metacling/src/TCling.cxx

```diff
@@ -3493,6 +3493,8 @@ std::string AtlernateTuple(const char *classname)
    guard << guard_name;
 
    std::ostringstream alternateTuple;
+   alternateTuple << "#ifdef __ROOTCLING__" << "\n";
```

Comment:
Why this code is executed twice? Can you set a breakpoint and show the stack traces for both times we enter here?